src/configuration/validators/vimrcValidator.ts

```diff
@@ -0,0 +1,22 @@
+import * as fs from 'fs';
+import { IConfiguration } from "../iconfiguration";
+import { IConfigurationValidator, ValidatorResults } from "../iconfigurationValidator";
+
+export class VimrcValidator implements IConfigurationValidator {
+  async validate(config: IConfiguration): Promise<ValidatorResults> {
+    const result = new ValidatorResults();
+
+    if (config.vimrc.enable && !fs.existsSync(config.vimrc.path) ) {
```

Comment:
The code in `vimrc.ts` has a lot more logic in checking the vimrc path. `VimrcImpl.expandHome`